src/main/java/seedu/address/logic/PomodoroManager.java

```diff
@@ -508,5 +512,21 @@ public void handleExit() {
         }
         float timeInMinutes = timeSeconds.floatValue() / 60;
         model.setPomodoroTimeLeft(timeInMinutes);
+
+        // Update statistics so far
+        // Update stats
+        updateStatistics(model);
```

Comment:
is this the part that handles the exiting?

Reply:
Yes it is!